### ISSUE TITLE
New branding | UI changes  

### ISSUE THREAD

**Issue by fannyBkane** [emfoundation/ce100-app#1223]
**New colours:**

**Cello Pink** `#FA3265`  >Accent/ functional colour
**Lavender Grey** `#585667` (replaces Grey 40667E) >Functional colour 

_Note about the Aqua blue :_ Although still part of the visual identity, it will become a secondary colour, and in the current state of the app, it should be replaced either by the Pink, or by a grey, depending on the context.

## Global changes (Mobile through to desktop)
- [x] Grey `#40667E` changes to `#585667`
- [x] Some buttons colour change to Pink `#FA3265` and the hover state changes to Lavender Grey with transition: 0.2 sec ease:
 - [x] 'Login' on login page
 - [x] 'Go' on Search and Search results pages
-  [x] 'Comment' button on challenge page colour changes to Lavender Grey with Hover state 10% darker and duration transition: default
- [x] On light backgrounds, Header 1 colour change to Blue `#003253 ` - org name on org profile, and the user's name on the user profile
- [x] Colour of Masthead on the Explore section (including Select tag page) and Challenge page changes to Blue/purple gradient
- [x] Tag style changes (default: white fill, grey text and Grey solid border / hover state: Grey fill, white text, transition: 0.2 sec ease
- [x] Colour of underline in tab component changes to Cello Pink (explore and org profile)
- [x] Modal header background changes to Lavender Grey
- [x] Background fills on forms mastheads change to Lavender Grey

## Mobile Top Nav:
- [x] Background changes to white
- [x] Links change to Midnight blue (`#003253`) 
- [x] Nav overlay background changes to Blue-Purple gradient
- [x] Log out link changes to Cello Pink

## Mobile Bottom Nav
- [x] Links and icons change to Lavender Grey
- [x] Active state bottom nav changes to Lavender Grey
- [x] Dropdown bottom nav background: changes to `#4F4E5D`

## Desktop Nav:
- [x] Background changes to white
- [x] Links change to Midnight blue (`#003253`) with hover state Cello Pink and a transition-duration *TBC**
- [x] Dropdown nav solid border changes to Cello Pink and Log out link changes to Cello Pink

## Login:
- [x] Image background added on the login page
- [x] Button colour and hover state changes (see above)

Preview screens here: https://invis.io/F3F94RCPK

**Comment by fannyBkane** [comment]
Hi @Cleop, the mobile login page is slightly different to desktop due to image constraint. The logo and tag line are on an image background, and the rest of the content on white one. Please see [here](https://projects.invisionapp.com/d/main#/console/12795181/272706370/preview)
![screen shot 2018-01-10 at 11 28 39](https://user-images.githubusercontent.com/20577339/34770639-89da7bba-f5f9-11e7-95f0-e681e36b19d2.png)

I've also changed the link style on the login page and mastheads of challenges based on accessibility issues that were raised in the call. made them all White or Pink (depending on context) but added an underline as seen on those examples:
![screen shot 2018-01-10 at 11 27 24](https://user-images.githubusercontent.com/20577339/34770506-51fab8f4-f5f9-11e7-8109-f4e653b20f6f.png)
![screen shot 2018-01-10 at 11 27 50](https://user-images.githubusercontent.com/20577339/34770508-52514c5a-f5f9-11e7-8a15-2578da9e2ead.png)

**Comment by Cleop** [comment]
>Some buttons colour change to Pink #FA3265 and the hover state changes to Lavender Grey with transition-duration TBC

Is this what you're describing for the hover state on all pink buttons?
Normal:
<img src="https://user-images.githubusercontent.com/16775804/34771637-2a8dffde-f5fd-11e7-81df-4674f611af1b.png" width=250px />

On hover:
<img src="https://user-images.githubusercontent.com/16775804/34771645-3196c874-f5fd-11e7-82d4-96007612dd25.png" width=250px />

**Comment by fannyBkane** [comment]
@Cleop in reference to this: 
> Image background added on the login page

Could you let me know if it is worth having the text+ links in the footer on a block colour in case the image background can't stretch to the height?
![screen shot 2018-01-10 at 12 26 51](https://user-images.githubusercontent.com/20577339/34772885-928fad54-f601-11e7-8e26-d0ce8e0cbefb.png)

**Comment by fannyBkane** [comment]
@Cleop 

> @fannyBkane Is this what you're describing for the hover state on all pink buttons?
That's right

> What do you mean by duration transition default?
It means no transition-duration.

**Comment by Cleop** [comment]
@fannyBkane - I think there's no need for the coloured block, at the moment the background image has plenty of height: 
![image](https://user-images.githubusercontent.com/16775804/34820273-f4538982-f6b7-11e7-845c-fc654a7f7555.png)
I haven't changed the logo and the statement underneath that yet but I think it will be fine.

**Comment by Cleop** [comment]
@fannyBkane - should the resources banner be lavender or gradient? All of the other small banners are lavender but the resources one is not a form so in theory it should be gradient?
![image](https://user-images.githubusercontent.com/16775804/34823208-7f0444fa-f6c1-11e7-89f9-77c299efa18a.png)

Should the Add New Challenge banner be lavender or gradient? It's a form so in theory should be lavender but it's big and all of the other big ones are gradient?
![image](https://user-images.githubusercontent.com/16775804/34823238-9796f742-f6c1-11e7-81e1-ff78d9e4b849.png)

I know that these are subject to change in the next issue if I get time to move on to that form bit so that's why I can't follow invision. But I thought I should check in case I don't get around to making those changes.

**Comment by Cleop** [comment]
This issue is now being put into please-test once the PR is merged. I want @fannyBkane to take a look at my question around the colour of mastheads. The other thing that it's worth highlighting is that when I was adding the gradient background I added the background on the search page. The bullet point for this is actually in the next issue. When I was adding it I realised I will have to work around the fact that the gradient is not opaque and as a result you can see what's behind it:
![image](https://user-images.githubusercontent.com/16775804/34823366-05a634aa-f6c2-11e7-996b-b964e1875c4f.png)

So don't worry, I'm aware of this issue but as it is technically for the next issue, I've not yet fixed it here.

**Comment by fannyBkane** [comment]
> About this
> ![screen shot 2018-01-11 at 12 22 40](https://user-images.githubusercontent.com/20577339/34825555-e3fd41a0-f6ca-11e7-837b-e6c29ac45c10.png)
> 

Can we keep the gradients in both cases? 

As a low priority it would be great to increase the masthead height of resources (making it consistent with other 'big' mastheads).

**Comment by fannyBkane** [comment]
> ![screen shot 2018-01-11 at 12 37 50](https://user-images.githubusercontent.com/20577339/34825926-69ffc95c-f6cc-11e7-9f98-6792b9522d9d.png)

That's right, they will be the same style as elsewhere on the app, except they have a 0.5 opacity.

**Comment by fannyBkane** [comment]
> ![screen shot 2018-01-11 at 12 40 30](https://user-images.githubusercontent.com/20577339/34825984-ad2383c2-f6cc-11e7-9b5f-ea0b78f0ae62.png)

Here I was referring to the cello logo (linking to the  walkthrough).

**Comment by fannyBkane** [comment]
> Some buttons colour change to Pink #FA3265 and the hover state changes to Lavender Grey with transition: 0.2 sec ease:

I don't think the transition has been added

**Comment by fannyBkane** [comment]
@Cleop @asimonw is it possible to remove the underline form 'Forgot your password' on the login page? otherwise LGTM
![screen shot 2018-01-11 at 18 09 36](https://user-images.githubusercontent.com/20577339/34839807-d59cff26-f6fa-11e7-9988-f2cfa866225b.png)

**Comment by asimonw** [comment]
Ok, @Cleop Is it possible to do what @fannyBkane suggests before I close this one?

**Comment by asimonw** [comment]
Actually, @Cleop one more thing: @fannyBkane sent through new bg images for the login screen (one for mobile, one for desktop). The sizes should be much smaller now. Note that the actual image has changed as well.

So that leaves:
+ remove the underline form 'Forgot your password' on the login page
+ replace login bg images on desktop and mobile

**Comment by asimonw** [comment]
@Cleop The image background doesn't really work as it does in the mockup (on desktop). It would work perfectly if you would remove the `min-height: 100vh` from `.container--login`. This breaks the forgotten password page, but you can instead set the `min-height: 100vh` on `.bg-login-ns`.

**Comment by asimonw** [comment]
@SimonLab This seems to be fine now. This should be in please test right?

**Comment by asimonw** [comment]
Thanks @Cleop looks great now!
